_includes/portfolio.html

```diff
@@ -64,7 +64,7 @@
 					<h3>{{ item.heading | markdownify | remove: "<p>" | remove: "</p>" }}</h3>
 					{% if item.meta %}<span class="meta">{{ item.meta }}</span>{% endif %}
 				</div>
-				<img src="{% if item.thumbnail %}{{ item.thumbnail }}{% endif %}" alt="" />
+				<img src="{% if item.categories contains "plugin-thumbnail" %}/plugins/images/thumbnails/plugin.png{% else %}{{ item.thumbnail }}{% endif %}" alt="" />
```

Comment:
So, I want to have the `plugin` category (now `plugin-thumbnail`) also force the usage of the built-in thumbnail.

If a user is submitting a plugin and wants to use this style, they should only have to specify
```yaml
categories:
  - plugin-thumbnail
```
and not have to *also* add

```yaml
thumbnail: /plugins/images/thumbnails/plugin.png
```
---
I'm not sure how I feel about this solution though.

I really don't like making a *page-specific* change to the generic `portfolio.html`.  I know that this is *only* for the Pico website, and therefore can be as hacky as we want it to be... but it still feels more like a workaround than a proper solution.  Any other solutions I could think of were much more complicated and didn't seem worth the effort.

Let me know if you have any ideas on how to handle this better.  :confused: